src/styles/main.scss

```diff
@@ -0,0 +1,58 @@
+.conteiner {
+  display: flex;
+  justify-content: center;
+  margin: 100px;
+}
+
+.arrow-minutes {
+  background-color: red;
+  width: 10px;
+  height: 150px;
+  border-top-left-radius: 5px;
+  border-top-right-radius: 5px;
```

Comment:
Combine it with [`border-radius` property](https://www.w3schools.com/cssref/css3_pr_border-radius.php)
Fix all cases